docs/UserGuide.md

```diff
@@ -14,27 +14,27 @@ ArtBuddy (AB) is a **desktop app for managing contacts, optimized for use via a
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `artbuddy.jar` from [here](https://github.com/AY2223S1-CS2103T-W11-3/tp/releases).
+2. Download the latest `artbuddy.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
```

Comment:
This here too, let's maybe edit it back to our TP.

Reply:
Thx will take not i will go read the instructions again before making any request so sorry